src/sparseml/pytorch/__init__.py

```diff
@@ -22,10 +22,21 @@
 from packaging import version
 
 
+def _default_opset(torch_version: version.Version) -> int:
+    if torch_version < version.parse("1.3"):
+        return 9
+    if torch_version < version.parse("1.10.0"):
+        return 11
+    return 13
+
+
 try:
     import torch
 
     _PARSED_TORCH_VERSION = version.parse(torch.__version__)
+
+    TORCH_ONNX_OPSET = _default_opset(_PARSED_TORCH_VERSION)
```

Comment:
let's move this to a helpers file, in general we don't perform much logic/set variables in our init files so it's not a place that is usually looked for